pkgs/development/python-modules/prometheus-flask-exporter/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "prometheus-flask-exporter";
```

Comment:
`fetchPypi` can't seem to find `prometheus-flask-exporter`.
It works with `prometheus_flask_exporter` with underscores instead of dashes.